packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -8,7 +8,7 @@
  * @flow
  */
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
This is needed because we are now importing types from React. `ElementRef`, and `AbstractComponent`.